sky/utils/schemas.py

```diff
@@ -1267,6 +1267,13 @@ def _get_controller_schema():
                         }]
                     },
                 },
+                'capabilities': {
+                    'type': 'array',
+                    'items': {
+                        'type': 'string',
+                        'case_insensitive_enum': ['compute', 'storage']
+                    },
+                },
```

Comment:
nit: should we extract this into a const, similar to `_LABELS_SCHEMA` and `_NETWORK_CONFIG_SCHEMA`.